src/infrastructure/config/settings_builder.py

```
@@ -222,6 +222,25 @@ def _build_content_handler_settings(value: Any) -> ContentHandlerSettings:
     )
 
 
+def _media_cache_ttl_seconds(media_cfg: Any) -> int:
+    # 与 schema slider 下界保持一致；防止绕过 AstrBot schema 直接构造出 0/负数 TTL。
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)